### PR TITLE
Added several security patches related to CVE-2022-29143

### PR DESCRIPTION
**Pull request by MikeyBronowski**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [x] Documentation
 - [ ] Build system

### CODE REVIEW

**Comment by niphlod** [comment]
@MikeyBronowski : bump the date at the top

**Comment by MikeyBronowski** [comment]
> @MikeyBronowski : bump the date at the top

Good spot. Done :)

**Review by niphlod — APPROVED** [review]

File: bin/dbatools-buildref-index.json [hunk 1]
```diff
@@ -4394,6 +4438,10 @@
             "CU": "CU16",
             "Version": "15.0.4223",
             "KBList": "5011644"
-        }
+        },
+        {
+            "Version": "15.0.4236",
+            "KBList": "5014353"
+        },
```
Comment (potatoqualitee) [inline bin/dbatools-buildref-index.json:4445]: thank you so much! a bunch of tests are failing and I think it's this last lil comma causing the problems

```suggestion
        }
```

Comment (potatoqualitee) [reply bin/dbatools-buildref-index.json:4445]: lemme see if this works -- that's line 4445, removing a comma

**Pull request merged**